src/type/payment-flow-params.ts

```diff
@@ -0,0 +1,4 @@
+export interface PaymentFlowParams {
```

Comment:
This type is also used in the frontend

Reply:
Ah good point. I will move to `beabee-common` in my next PR :+1: